subsys/fs/fcb/fcb_elem_info.c

```diff
@@ -63,14 +65,50 @@ fcb_elem_crc8(struct fcb *fcb, struct fcb_entry *loc, uint8_t *c8p)
 	return 0;
 }
 
+int
+fcb_elem_endmarker(struct fcb *_fcb, struct fcb_entry *loc, uint8_t *c8p)
+{
```

Comment:
start with the check:
```
        if (loc->fe_elem_off + 2 > loc->fe_sector->fs_size) {
		return -ENOTSUP;
	}
```

Reply:
In `fcb_elem_crc8()` this is then no longer required.

Reply:
true, I kept it as `fcb_elem_crc8` is called other places as well (`fcb_elem_info`), but here it is guarded by calling `fcb_elem_endmarker` first anyway. Will update :)